application/src/main/resources/locale/blueprint/horizons/description.csv

```diff
@@ -1,176 +1,176 @@
+blueprint.description.horizons.engineer_c1a_learn,Learn about: Reach level 3 and 33% reputation with Juri Ishmaak.,Learn about: Reach level 3 and 33% reputation with Juri Ishmaak.,Erfahren Sie mehr über: Ruf von Level 3 und 33% Ruf bei Juri Ishmaak.,Aprenda sobre: alcanzar la reputación de nivel 3 y 33% con Juri Ismaak.,Découverte: Atteindre la réputation de niveau 3 et 33% avec Juri Ishmaak.,Aprenda sobre: Alcance Nível 3 e 33% de reputação com Juri Ismaak.,Узнайте о репутации уровня 3 и 33% с Юри Ишмааком.,
+blueprint.description.horizons.engineer_c1a_unlock,Initial contract: 1M Credits worth of Combat Bonds.,Initial contract: 1M Credits worth of Combat Bonds.,Erstvertrag: 1M Credits im Wert von Kampfanleihen.,Contrato inicial: 1 millones de créditos de bonos de combate.,Contrat initial: 1M Crédits de primes de combat.,Contrato inicial: 1M créditos no valor de títulos de combate.,Первоначальный контракт: 1 млн кредитов на боевые облигации.,
+blueprint.description.horizons.engineer_c1b_invite,Gain access: Gain rank Midshipman or higher with the Federal Navy.,Gain access: Gain rank Midshipman or higher with the Federal Navy.,Zugriff erhalten: Gewinnrang Midshipman oder höher mit der Bundesmarine.,Obtenga acceso: Gane Rank Midshipman o superior con la Marina Federal.,Invitation: Gagnez du rang Matelot ou plus avec la marine fédérale.,Ganhe acesso: Ganho Rank Midshipman ou superior com a Marinha Federal.,Получите доступ: выиграйте ранжирование мичмана или выше с федеральным флотом.,
+blueprint.description.horizons.engineer_c1b_learn,Learn about: Reach level 3 and 33% reputation with Juri Ishmaak.,Learn about: Reach level 3 and 33% reputation with Juri Ishmaak.,Erfahren Sie mehr über: Ruf von Level 3 und 33% Ruf bei Juri Ishmaak.,Aprenda sobre: alcanzar la reputación de nivel 3 y 33% con Juri Ismaak.,Découverte: Atteindre la réputation de niveau 3 et 33% avec Juri Ishmaak.,Aprenda sobre: Alcance Nível 3 e 33% de reputação com Juri Ismaak.,Узнайте о репутации уровня 3 и 33% с Юри Ишмааком.,
+blueprint.description.horizons.engineer_c1b_unlock,Initial contract: 50 units of Aberrant Shield Pattern Analysis.,Initial contract: 50 units of Aberrant Shield Pattern Analysis.,Erstvertrag: 50 Einheiten einer aberranten Schildmusteranalyse.,Contrato inicial: 50 unidades de análisis de patrones de escudo aberrante.,Contrat initial: 50 unités de données d’analyse de bouclier aberrante.,Contrato inicial: 50 unidades de análise de padrão de escudo aberrante.,Первоначальный контракт: 50 единиц анализа аберрантного щита.,
```

Comment:
unless there is any significant difference, can you use the ' character for quotes? otherwise it is used in different ways throughout the app and I rather keep it consistent. also the quote needs to be escaped by another quote so: d''analyse

Reply:
Ok I escaped the quotes where I found they were not. 
I'm not sure of what other characters you have found. I saw some Ascii 148 or 149, not sure, but they were already present in previous description that I didn't change. Do those have to be changed ?

Reply:
for example `Invitation: Gagner le rang d’éclaireur ou supérieur en exploration` contains a ’ instead of a '
it can be hard to search for them in the browser since they are both treated as a single quote when you search for them. if you push your fixes, I can look for them. Right now due to the comma's at the end the entire file shows as changed in the diff